metron-platform/metron-elasticsearch/src/main/java/org/apache/metron/elasticsearch/dao/ElasticsearchMetaAlertDao.java

```diff
@@ -275,13 +275,13 @@ protected Document buildCreateDocument(MultiGetResponse multiGetResponse, List<S
         alertList.add(response.getSource());
       }
     }
-    metaSource.put(ALERT_FIELD, alertList.toArray());
+    metaSource.put(ALERT_FIELD, alertList);
```

Comment:
@justinleet  "The alerts field can actually come as two different types List<Object> vs Object[] during calculation of meta score fields."


Your changes here seem simple enough.  Before we used arrays, now using Lists.  But I am not following how this solves the problem you describe in JIRA.  

Where is the code that would handle/transform/standardize items that come in as either Array or List?

Reply:
If you look at the first two commits, they were originally to do the JIRA as noted (handle both).  The solution of the last commit is to just not use arrays at all. In actuality though, both constructions of the alert field are controlled by the DAO itself, so there's no reason to use arrays at all.

Basically, this PR solves the underlying problem (The two paths used different types), rather than the symptom (That two different types went into the same method).

Reply:
Updated the PR description to make the reasoning more obvious to anyone glancing there first.